packages/xod-project/src/internal/patchPathUtils.js

```diff
@@ -1,6 +1,15 @@
 import * as R from 'ramda';
 import * as CONST from '../constants';
 
+const EXPANDED_PATCH_NAME_SUFFIX = '-$';
+
+export const getExpandedPatchPath = R.curry(
+  (arityLevel, originalPath) => [
+    originalPath,
+    EXPANDED_PATCH_NAME_SUFFIX,
+    arityLevel,
+  ].join('')
+);
```

Comment:
The `packages/xod-project/src/internal/patchPathUtils.js` gives no idea we’re talking about variadics here. So I’d call the function more verbose: `getExpandedVariadicPatchPath